boilerplate/openshift/golang-osd-operator/standard.mk

```diff
@@ -108,9 +108,16 @@ coverage:
 	boilerplate/openshift/golang-osd-operator/codecov.sh
 
 .PHONY: test
-test: gotest yaml-validate
+test: gotest validate-olm-deploy-yaml
 
-.PHONY: yaml-validate
-yaml-validate:
+.PHONY: install-pyyaml
+install-pyyaml:
 	python3 -mpip install pyyaml
```

Comment:
This doesn't work in CI because it tries to install into the real py3 dist, which is owned by root.

Adding `--user` is supposed to be the way around that, but that *also* doesn't work because the CI user's `$HOME` isn't set, so it tries to install in `/.local`.

I'm working on the PR to do set up the venv so we can do these things there.

Reply:
Yeap this is covered in #42